src/test/java/org/opensearch/security/ssl/SecuritySSLReloadCertsActionTests.java

```diff
@@ -294,6 +317,29 @@ public void testReloadTransportCertDifferentTrustChain_noSkipDnValidationFail()
         );
     }
 
+    @Test
+    public void testReloadTransportCertDifferentTrustChain_defaultSettingValidationFail() throws Exception {
+        updateFiles(defaultCertFilePath, pemCertFilePath);
+        updateFiles(defaultKeyFilePath, pemKeyFilePath);
+        initTestCluster(pemCertFilePath, pemKeyFilePath, pemCertFilePath, pemKeyFilePath, true, true, null);
```

Comment:
I know that `true` is the default, but to really test default behavior shouldn't this test pass `null` for both `httpEnforceReloadDnVerification` and `transportEnforceReloadDnVerification`?

Reply:
Agreed, have updated both default test cases